lib/util/patterns/L.js

```diff
@@ -0,0 +1,10 @@
+/**
```

Comment:
Did you grab this from JSCS? If so, we need to be sure to include proper licensing information here.

Also, can you rename this file? It doesn't follow our conventions of all lowercase and dash-cased. Maybe `letters.js`?